src/base/portal.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { useRef, useMemo, useState } from 'react';
+import React, { useRef, useMemo } from 'react';
 import ReactDOM from 'react-dom';
 
 const PORTAL_ID = 'rmwcPortal';
 
 export type PortalPropT = Element | string | boolean | undefined | null;
 
 export function Portal() {
-  const el = useRef(document.createElement('div'));
+  const el = useRef<HTMLDivElement | null>(null);
```

Comment:
This was probably being done for typesafety so that el was initialized to a div, but this really didn't make any practical sense. So initializing this to null.